src/wallet/tx_builder.rs

```diff
@@ -646,14 +649,22 @@ impl<'a, B, D: BatchDatabase> TxBuilder<'a, B, D, DefaultCoinSelectionAlgorithm,
 }
 
 /// Ordering of the transaction's inputs and outputs
-#[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Hash, Clone, Copy)]
+#[derive(Clone)]
 pub enum TxOrdering {
     /// Randomized (default)
     Shuffle,
     /// Unchanged
     Untouched,
     /// BIP69 / Lexicographic
+    #[deprecated = "BIP69 does not improve privacy as was the intention of the BIP"]
     Bip69Lexicographic,
+    /// Provide custom comparison functions for sorting
+    Custom {
+        /// Transaction inputs sort function
+        input_sort: Arc<dyn Fn(&TxIn, &TxIn) -> Ordering>,
```

Comment:
Very minor nit, but this confused me for a second: I would just import `std::cmp` and then here use `cmp::Ordering`.

For a second I confused `Ordering` with `TxOrdering` and I couldn't figure out what was going on.